public/docs/_examples/forms/dart/web/main.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:angular2/bootstrap.dart';
```

Comment:
I think Angular has to because of typescript weirdness – not required at all unless you're trying to avoid a conflict